### ISSUE TITLE
RenameClassRector uses FQCN despite importing the class

### ISSUE THREAD

**Issue by keulinho** [rectorphp/rector#1564]
I ran rector with the  symfony43 level, but when renaming the classes rector uses the FQCN in the import statement and in the Code. It also uses a trailing `\`. 

![image](https://user-images.githubusercontent.com/15930605/58934389-9eae0a80-876a-11e9-9acf-5162ca1342b8.png)

### Ecpected Behaviour
The expected behaviour would be that it uses the FQCN in the import statement but without the leading `\`. And inside the code it should use the simple class name if the class is already imported.
In this case the two changed lines would become
```php
use Symfony\Component\HttpKernel\Event\ExceptionEvent;
public function onKernelException(ExceptionEvent $event)
```

I've found the issue https://github.com/rectorphp/rector/issues/1437 where the behaviour of always using the FQCN is desired, but just if the class is not already imported.
Maybe some changes where made there, that introduced this issue.

**Comment by TomasVotruba** [comment]
The best ways to solve this is using coding standard to handle your imports.
Using FQN by default is to allow them import or keep FQN - some people prefer first, some second. And it's harder to FQNize than import in coding standard rules.

tl;dr; AST sucks in coding standard, and always will. It doesn't make sense to rewrite EasyCodingStandard, PHP CS Fixer or CodeSnffier. Just use it.

See this comment for deeper answer: https://github.com/rectorphp/rector/issues/1456#issuecomment-495296417

I recommend using option `--with-style`: https://github.com/rectorphp/rector#how-to-apply-coding-standards
